components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SubscriptionValidationSQLConstants.java

```diff
@@ -19,7 +19,7 @@
 package org.wso2.carbon.apimgt.impl.dao.constants;
 
 public class SubscriptionValidationSQLConstants {
-
+    public static final String IS_SYSTEM_APP_ATTRIBUTE = "isSystemApp";
```

Comment:
Move to APIConstants

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/12101/commits/202fb123e6ff5128e5b52ec84b71d792457a2119